test/UnitTestParquetCpp.chpl

```diff
@@ -3,16 +3,34 @@ use UnitTest;
 use TestBase;
 
 proc testReadWrite(filename: c_string, dsetname: c_string, size: int) {
-  extern proc c_readColumnByName(filename, chpl_arr, colNum, numElems);
+  extern proc c_readColumnByName(filename, chpl_arr, colNum, numElems, errMsg): int;
   extern proc c_writeColumnToParquet(filename, chpl_arr, colnum,
-                                     dsetname, numelems, rowGroupSize);
+                                     dsetname, numelems, rowGroupSize,
+                                     errMsg): int;
+  extern proc c_free_string(a);
+  extern proc strlen(a): int;
+  var errMsg: c_ptr(uint(8));
+  defer {
+    c_free_string(errMsg);
+  }
+  var causeError = "asdasdasd":c_string;
```

Comment:
Maybe this should have a better default value?  (Just now realizing I'm in the Unit test, so no big deal)